packages/client-whatsapp/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import { elizaLogger } from "@ai16z/eliza";
```

Comment:
Not sure why these whatsapp changes are included

Reply:
looks like upstream changes from before idk